templates/rhpam73-authoring-ha.yaml

```diff
@@ -1302,15 +1441,30 @@ objects:
             value: "openwire"
           - name: AMQ_GLOBAL_MAX_SIZE
             value: ${AMQ_GLOBAL_MAX_SIZE}
-          image: "${AMQ_IMAGE_STREAM_NAME}"
-          imagePullPolicy: Always
-          readinessProbe:
-            exec:
-              command:
-              - "/bin/bash"
-              - "-c"
-              - "/opt/amq/bin/readinessProbe.sh"
-          name: ${APPLICATION_NAME}-amq
+          - name: AMQ_REQUIRE_LOGIN
+          - name: AMQ_DATA_DIR
+            value: /opt/amq/data
+          - name: AMQ_DATA_DIR_LOGGING
+            value: "true"
+          - name: AMQ_CLUSTERED
+            value: "true"
+          - name: AMQ_REPLICAS
+            value: "0"
+          - name: AMQ_CLUSTER_USER
+            value: ${APPFORMER_JMS_BROKER_USER}
+          - name: AMQ_CLUSTER_PASSWORD
+            value: ${APPFORMER_JMS_BROKER_PASSWORD}
+          - name: AMQ_EXTRA_ARGS
```

Comment:
I copied from the original template, but I understood why they added envs with no value because according to API docs it defaults to an empty string. Having that said, the AMQ team decided to not add the value attribute as the default is already an empty string. If you disagree with that, I can add it anyway.

Reply:
not sure if I follow you, ENV with no value and no ENV produces the same result?

Reply:
Actually, what I meant is: ENV with no value and ENV with value set as empty string produces the same result.